KunligiOnline/kunligiconnect
# Add basic react components

## Description:
- The app had no components. I added basic components across 4 pages: Home.tsx(where users will choose the type of conversation), Login.tsx(where users will start to login), Chat.tsx(where users will message each other), and Loading.tsx(our loading page)
## Changes I Made:
- `src/App.tsx`:
- `src/components/chat`:
- `src/components/home`:
- `src/components/loading`:
- `src/components/chat`:
    - Chat.tsx:
    - Messages.tsx:
    - MessageInput.tsx:
- `package.json`: Added react-router-dom and types for react-router-dom
## How to Test:
- npm install
- npm run start
- Check the following URLs for components:
  - localhost:3000/
  - localhost:3000/home
  - localhost:3000/loading
  - localhost:3000/chat
